sci-calculators/bc-gh/bc-gh-6.2.2.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Implementation of POSIX bc with GNU extensions"
+HOMEPAGE="https://git.yzena.com/gavin/bc"
+SRC_URI="https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz"
```

This exceeds the 80-char limit. Wrap it, so at least it will fit on the screen for those of us who use shorter tabs.

```suggestion
SRC_URI="
	https://github.com/gavinhoward/bc/releases/download/${PV}/bc-${PV}.tar.xz
"
```